### DISCUSSION TITLE
Processing bruker .d files with TIMS on in MS-Dial > Version 5.

### DISCUSSION THREAD

**Discussion by dschief001** [systemsomicslab/MsdialWorkbench#471] [Q&A]
Hi!

In the newer versions, I don't get my bruker .d files to get analyzed correctly, tried lately 3 times with the lastest MS-DIAL release  MSDIAL.v5.5.241113-net48 on a windows 10 pro machine.
* files couldnt be loaded through the browse dialog (no .d filter available),
* directly pulling the files into the MS-DIAL window got them registered, and the individual files where processed, the process then died during alignment/ (when there is only one bar before it normally finishes.
* conversion to ibf file format (with converter shipped with MS-DIAL 4.6, conversion worked), but also those files, during alignment the new ms dial stopped responding.

Hence I would love to know if this is an issue on my fault, of  wrongly prepared files, or if this is a common MS-Dial issue?

Thank you very much for your reply.

Jakob

**Comment by marlenethaitumu** [comment]
Hi @dschief001 I have been having the same problem trying to process data with TIMS ON.  I'm not even able to open the ibf converter.  Which program open the ibf converter?  Were you able to get this issue fixed.  Please let me know.  Thanks!
